source/content/guides/secrets/04-use-cases.md

```diff
@@ -0,0 +1,274 @@
+
+    ```bash
+    composer require your-organization/your-package-name
+    ```
+
+1. Commit your changes and push to Pantheon.
+
+</Tab>
+
+</TabList>
+
+## Mechanism 2: HTTP Basic Authentication
+
+You may create a `COMPOSER_AUTH json` and make it available via the `COMPOSER_AUTH` environment variable if you have multiple private repositories on multiple private domains.
+
+Composer has the ability to read private repository access information from the environment variable: `COMPOSER_AUTH`. The `COMPOSER_AUTH` variables must be in a [specific JSON format](https://getcomposer.org/doc/articles/authentication-for-private-packages.md#http-basic).
+
+**Format example:**
+
+```bash
+#!/bin/bash
+
+read -e COMPOSER_AUTH_JSON <<< {
+    "http-basic": {
+        "github.com": {
+            "username": "my-username1",
+            "password": "my-secret-password1"
+        },
+        "repo.example2.org": {
+            "username": "my-username2",
+            "password": "my-secret-password2"
+        },
+        "private.packagist.org": {
+            "username": "my-username2",
+            "password": "my-secret-password2"
+        }
+    }
+}
+EOF
+
+terminus secret:site:set ${SITE_NAME} COMPOSER_AUTH ${COMPOSER_AUTH_JSON} --type=env --scope=ic
+```
+
+# ☞ Accessing secrets from your codebase
+
+## Introduction
+
+It is only possible to read secrets from the Drupal or WordPress site. Secrets are created and modified via the [Terminus Secrets Manager Plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin).  Modifying secrets from within the application itself is not supported.
+
+This SDK will only read secrets with scope web. Secrets get cached in the server for 15 minutes so you should wait (at most) that time if you modified your site secrets.
+
+Note: this also applies to quicksilver scripts
+
+## Mechanism 1: SDK (composer package)
+
+This PHP library is the recommended approach for accessing secrets from the Secrets Service.
+
+Use composer to install it:
+
+```
+composer require pantheon-systems/customer-secrets-php-sdk
+```
+
+### Usage
+
+In your PHP code, do the following:
+
+```
+use PantheonSystems\CustomerSecrets\CustomerSecrets;
+
+$client = CustomerSecrets::create()->getClient();
+$secret = $client->getSecret('foo');
+$secret_value = $secret->getValue();
+
+// You could also get all of your secrets like this:
+$secrets = $client->getSecrets();
+```
+
+See the included classes and internal documentation for more examples and use cases.
+[https://github.com/pantheon-systems/customer-secrets-php-sdk?tab=readme-ov-file#usage](https://github.com/pantheon-systems/customer-secrets-php-sdk?tab=readme-ov-file#usage)
+
+## Mechanism 2: get_pantheon_secrets
+
+A `pantheon_get_secrets()` function is provided for sites that aren't built with Composer.
```

Comment:
```suggestion
For sites that do not use Composer, the function `pantheon_get_secret()` may be used to fetch the value of a single secret.
```

Reply:
This merge conflicted with a change I made but I changed this language as suggested.